src/removeNPMAbsolutePaths.js

```diff
@@ -44,7 +44,12 @@ function processFile(filePath, opts) {
       }
 
       Object.keys(obj).forEach((key) => {
-        if (key[0] === '_') {
+        if (!opts.userSpecifiedKeys) {
```

Comment:
This duplicated code unnecessarily.
Also, as mentioned above, `opts.userSpecifiedKeys` is not really necessary. You can just check if `opts.keys` is defined.
Something like:
```
        const shouldBeDeleted = opts.fields ? (opts.fields.indexOf(key) !== -1) : (key[0] === '_');
        if (shouldBeDeleted) {
          delete obj[key];
          writeFile = true;
        }
```